### ISSUE TITLE
Add support for problem_type in config.json

### ISSUE THREAD

**Issue by KiterWork** [xenova/transformers.js#248]
Transformers.js is missing support for "problem_type": "multi_label_classification", which is specified in config.json. Just an easy way to track its addition to the library. 

```js
async function importTransformers(modelName) {
        let { pipeline, env } = await import('@xenova/transformers')

        env.localModelPath = './models/'
        env.allowRemoteModels = false
        
        let pipe = await pipeline("text-classification", `${modelName}`, {quantized: false})
        
        console.log(await pipe("I like you, i love you", {topk: null}))

}

importTransformers("unitary/toxic-bert")
```

Returns output1
```js
async function query(data) {
	const response = await fetch(
		"https://api-inference.huggingface.co/models/unitary/toxic-bert",
		{
			headers: { Authorization: "Bearer token" },
			method: "POST",
			body: JSON.stringify(data),
		}
	);
	const result = await response.json();
	return result;
}

query({"inputs": "I like you. I love you"}).then((response) => {
	console.log(response);
});
```
Returns output2

Output1 (wrong model output)
```js
[
  { label: 'toxic', score: 0.5042400360107422 },
  { label: 'obscene', score: 0.11989585310220718 },
  { label: 'insult', score: 0.10829909890890121 },
  { label: 'threat', score: 0.09879514575004578 },
  { label: 'identity_hate', score: 0.09252029657363892 },
  { label: 'severe_toxic', score: 0.07624954730272293 }
]
```
Output2 (correct output)

```js
[
  [
    { label: 'toxic', score: 0.0007729045464657247 },
    { label: 'obscene', score: 0.0002012521290453151 },
    { label: 'insult', score: 0.00017819840286392719 },
    { label: 'identity_hate', score: 0.000157709582708776 },
    { label: 'threat', score: 0.0001559475640533492 },
    { label: 'severe_toxic', score: 0.00012961401080247015 }
  ]
]
```

**Comment by xenova** [comment]
Thanks for the request! I've opened a [PR](https://github.com/xenova/transformers.js/pull/249) for this feature, and I'll merge into main (+ make a new release) shortly.

**Comment by xenova** [comment]
@KiterWork now available in [v2.5.2](https://www.npmjs.com/package/@xenova/transformers/v/2.5.2)!
